tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2277,7 +2277,7 @@ private boolean wantRevertToBaseline(String xpath, String vxmlValue) {
                 Output<String> inheritancePathWhereFound = new Output<>();
                 Output<String> localeWhereFound = new Output<>();
                 baselineFileResolved.getBaileyValue(xpath, inheritancePathWhereFound, localeWhereFound);
-                if (localeID.equals(inheritancePathWhereFound.value) || xpath.equals(inheritancePathWhereFound.value)) {
+                if (localeID.equals(localeWhereFound.value) || xpath.equals(inheritancePathWhereFound.value)) {
```

Comment:
I fixed the bug on this line; unfortunately the unwanted change in af is still there...

Reply:
Remember the note from the VXML best 43 data.
>Note: We can assume that – and test that — any locale/path that didn’t permit voting in baseline MUST have the same resolved value as in baseline. That means the only items that should differ are: …

So any value, ^^^ or not, that is in a disallowed path must be the same as in main. I think you need to modify your flow of control to ignore any value coming from vxml that is on a disallowed path as the first step.

Reply:
that would seem to differ from these criteria:
```
            /**
             * 	Revert each L2+ locale/path VXML value of ↑↑↑ to the baseline value, where all the following criteria are met:
             * 	1	The baseline value is a hard value (can use unresolved file for this)
             * 	2	The survey tool did not allow changes in the locale/path in v43
             * 	    ▪ That is, other than allowed locales with Person Names, Turkey, missing, and errors
             *    	▪ NOTE: we should be able to check missing/errors by running the CheckCLDR tests on the baseline data. Most locales only had PN & Turkey, but some had errors or missing values.
             * 	3	The bailey value is fetched from a different path AND from an ancestor locale.
             * 	    ▪ Ideally we would look at intervening paths also, but I think this is sufficient.
             */
```
-- so I'll try to revise the statement of the criteria as well as the code...

Reply:
Yes, we modified the criteria (with respect to L2+, for example); that should be in your notes from the meeting. I just updated https://docs.google.com/document/d/1F_WODFRIgBgn2CGHVyPpDi2iMMqhFzHL3JCIaaPHurI/edit#heading=h.2j96gaxk85s1 so that there would not be confusion.

As to:
> As it turns out, "modify -fR" finds the baseline value is "↑↑↑", NOT "{0} swaartekrag" 
> -- at least if the unresolved file is used -- af.xml has "↑↑↑"
…
>I fixed the bug on this line; unfortunately the unwanted change in af is still there...

_I will go through what I think should be happening. You should check this to see where it is failing in the new CLDRModify code._

In main we have the following (just checked) for locale af.xml
```
<unitLength type="long">
	<unit type="acceleration-g-force">
		<displayName>swaartekrag</displayName>
		**<unitPattern count="one">{0} swaartekrag</unitPattern>**
		<unitPattern count="other">{0} swaartekrag</unitPattern>
	</unit>
```
So the value for main ({0} swaartekrag) is correct as is.

The path is //ldml/units/unitLength[@type="long"]/unit[@type="acceleration-g-force"]/unitPattern[@count="one"]

That path was populated, was in a CLDR-TC locale, but not one of the limited submission paths (personNames, etc). The DraftStatus was **approved** in v42 **and** in main (ie, no draft attribute), so it was not missing. It also does not have an error.

Thus the path disallows entry in the Survey tool. 

You are merging into main, from vxml. 

Because the path does not allow entry, you are ignoring the vxml value, and using the main value.

Reply:
When I wrote "As it turns out, "modify -fR" finds the baseline value is "↑↑↑", NOT "{0} swaartekrag" ..." it was simply due to a mistake.

The other good news is this: the last push revised -fR so v43-disallowed paths all get values from baseline, not vxml, as (I think) you proposed. The af "acceleration-g-force" problem is now gone.